model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -97,6 +97,16 @@ class OrtConfigurationTest : WordSpec({
                     val cdStorage = this["clearlyDefined"]
                     cdStorage.shouldBeInstanceOf<ClearlyDefinedStorageConfiguration>()
                     cdStorage.serverUrl shouldBe "https://api.clearlydefined.io"
+                    
```

Comment:
Detekt complains about trailing spaces in this line.

Reply:
Fixed